app/workers/evss/disability_compensation_form/submit_form8940.rb

```diff
@@ -78,7 +78,7 @@ def upload_to_vbms
 
       def client
         @client ||= if Flipper.enabled?(:disability_compensation_lighthouse_document_service_provider)
-                      # TODO: create client from lighthouse document service
+                      BenefitsDocuments::Service.new(submission.auth_headers)
```

Comment:
Similar situation to https://github.com/department-of-veterans-affairs/vets-api/pull/13576/files#r1327579513